aiida/cmdline/params/options/__init__.py

```diff
@@ -585,8 +585,10 @@ def decorator(command):
     '-k', '--keys', type=click.STRING, cls=MultipleValueOption, help='Filter the output by one or more keys.'
 )
 
-DEBUG = OverridableOption(
-    '--debug', is_flag=True, default=False, help='Show debug messages. Mostly relevant for developers.', hidden=True
+VERBOSITY = VerbosityOption(
+    '-v',
+    '--verbosity',
+    metavar='LEVEL',
```

Comment:
~~Shouldn't this get a help string?~~ never mind, see that it is done dynamically in the `VerbosityOption` constructor